templates/designToClassBody.jinja

```diff
@@ -0,0 +1,145 @@
+{# © Copyright CERN, 2020.                                                       #}
```

Comment:
My another general comment (applies to both .jinja files) is non-uniform whitespacing.
Please could you convert tabs to spaces and consider 2 spaces for an indent level, like it is in all remaaining quasar jinja transforms?
(we can put this comment in the docs as well)

Reply:
Alright, I am going for it.